### PR TITLE
Real-to-Virtual version 1.0 Documentation

### LINKED ISSUE #90: Add Real-to-Virtual documentation
Add Real-to-Virtual documentation to the RTD site. [This issue](https://github.com/monoDriveIO/r2v-deployment/issues/33) from the `r2v-deployment` repo is related.

### PR DESCRIPTION
**Pull request by bradleynull**
This implements a couple quick start guides for real-to-virtual. There's a data collection quick start which is intended to go through all the steps necessary to set up sensors, collect data, and parse it into the KITTI format. There's also a data processing quick start which roughly goes through the data processing pipeline. Since the processing pipeline is still a WIP, the documentation will need to be updated once the final products is complete.

Closes #90

### CODE REVIEW

File: docs/r2v/detailed_guide.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+#### Data Logger Recording:
+1. Open the **r2v.lvprj** file on the R2V folder
+
+1. Double-click on the **main.vi**
+
+1. Click on the arrow to run the main.vi
+
+1. Select the path where the logs are going to be stored using the “Directory 
+to Log” control: 
+    a. The program will create the following directory structure to store your data:
```
Comment (citlaligm) [inline docs/r2v/detailed_guide.md:138]: Missing a new line 
image

File: docs/r2v/detailed_guide.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+    **NOTE:** By default there is a 5 second delay after you hit record to allow the sensors to initialize prior to recording.
+
+1. When ready to stop recording data, click the button Stop Rec.
+
+1. The system will wait for you to hit the button Configure and Record. 
+
+1. When you are ready to finish collecting data, click the EXIT button, this will disconnect the GNSS and close the open connections.
+
+    **NOTE:** If the the LabView “Stop” button is pressed instead of the R2V EXIT button, the entire application will need to be restarted in order to record again.
```
Comment (citlaligm) [inline docs/r2v/detailed_guide.md:156]: extra _"the"_

File: docs/r2v/data_collection_quick_start.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+For the monoDrive Real-to-Virtual hardware, the original spherical camera data
+must be calibrated for proper field-of-view prior to parsing in order to crop
+out static objects. The orignal shows a good portion of the data collection 
+vehicle:
+
+<div class="img_container">
+    <img class='lg_img' src="../imgs/kitti_data_full_size_image.png"/>
+</div>
+
+The data collection software provides a tool to determine the optimal
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:38]: Yea, I  figured that was the case, but I wanted to get this here to show that something existed or will exist. I'd vote to leave it in for now.

File: docs/r2v/data_collection_quick_start.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+* LiDAR - [Velodyne Puck Hi-Res](https://velodynelidar.com/products/puck-hi-res/) or better
+* Camera for scene generation - [FLIR Blackfly](https://www.flir.com/products/blackfly-s-usb3/?model=BFS-U3-63S4C-C)
+* Camera for dynamic actors - [FLIR Ladybug 5+](https://www.flir.com/products/ladybug5plus/)
+* GNSS or GPS + IMU - [Xsens MTi-G-710](https://shop-us.xsens.com/shop/mti-g-710/mti-g-710-gnssins-2a8g4) or better
+
+The monoDrive Real-to-Virtual hardware provides the proper sensor suite and 
+transforms in a package that can be easily mounted to a roof mount:
+
+<div class="img_container">
+    <img class='lg_img' src="../imgs/real_to_virtual_hardware_side_view.png"/>
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:17]: This image is in the repo and appears to work for me: https://github.com/monoDriveIO/documentation/blob/r2v_quickstart/docs/r2v/imgs/real_to_virtual_hardware_side_view.png

File: docs/r2v/data_collection_quick_start.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+#### GNSS Calibration
+
+In order to achieve an accurate heading from the Xsens GNSS, the magnetic field
+mapper tool must be  used to calibrate for the local magnetic bias. The 
+[Magnetic Calibration Manual](https://www.xsens.com/hubfs/Downloads/Manuals/MT_Magnetic_Calibration_Manual.pdf) Section 3.3 specifies the steps necessary
+to calibrate. 
+
+1. Open that Xsens Magnetic Field Mapper software
+1. Select "Write results to Motion Tracker non-volatile memory"
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:82]: Thanks for looking that up. I was just going by what the Xsens documentation was saying.

Fixed to match.

File: docs/r2v/data_collection_quick_start.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+1. Once the device is found, click next "Start" to begin calibration
+1. Drive the vehicle in **two consecutive circles** then click "Process"
+1. If result state "Acceptable" or better, select "Write to selected devices"
+
+### Data Collection
+
+After calibration, **do not power off the hardware**. This will preserve all
+calibration parameters. To collect data:
+
+1. Click the "Run" arrow on the top-left of the monoDrive LabView data logger application. The data logger will connect to all sensors and ensure they are functioning as expected.
```
[the remark this answers is not in the document]
Comment (bradleynull) [reply docs/r2v/data_collection_quick_start.md:93]: Fixed for all r2v documentation.

File: docs/r2v/detailed_guide.md (hunk 1 above)
Comment (bradleynull) [reply docs/r2v/detailed_guide.md:138]: Screenshot from 2020-04-20 12-03-27
Hmmm this is rendering correctly for me... let me know after you pull if it still doesn't look right to you.

File: docs/r2v/detailed_guide.md (hunk 2 above)
Comment (bradleynull) [reply docs/r2v/detailed_guide.md:156]: Good catch! Fixed.

**Comment by bradleynull** [comment]
@citlaligm Thanks for looking through this! I addressed all your comments and its ready for another look.

I've also added an additional page about using the UE level when complete. Please give that a look too. Thanks!

File: docs/r2v/data_collection_quick_start.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+must be calibrated for proper field-of-view prior to parsing in order to crop
+out static objects. The orignal shows a good portion of the data collection 
+vehicle:
+
+<div class="img_container">
+    <img class='lg_img' src="../imgs/kitti_data_full_size_image.png"/>
+</div>
+
+The data collection software provides a tool to determine the optimal 
+field-of-view in both the horizontal and vertial dimensions.
```
Comment (citlaligm) [inline docs/r2v/data_collection_quick_start.md:39]: Just see this other typo.

**Pull request merged**